docs/.vitepress/config.ts

```diff
@@ -40,6 +40,7 @@ const Integrations: DefaultTheme.NavItemWithLink[] = [
   { text: 'PostCSS', link: '/integrations/postcss' },
   { text: 'ESLint', link: '/integrations/eslint' },
   { text: 'VSCode extension', link: '/integrations/vscode' },
+  { text: 'WebStorm plugin', link: '/integrations/webstorm' },
```

Comment:
It should work in any JetBrains IDE, I'm using IntelliJ Ultime Edition and it is also working. Maybe we can switch the text and the md file to `JetBrains IDE Plugin` and `jetbrains` respectivelly

Reply:
?

Reply:
ok

Reply:
I mean, it should work with any JetBrains IDE since share the logic between IDE, and it should work.

Check Vitest ide entry https://vitest.dev/guide/ide.html#jetbrains-ide, in the history and you'll see the community plugin: https://github.com/vitest-dev/vitest/commit/30306eec3a1ed361f12a3d032cff1517c11d0873 (IntelliJ/WebStorm)

Reply:
done